### DISCUSSION TITLE
Deploying our Contract to Testnet or Live Network with Foundry and Alchemy (Error with the command)

### DISCUSSION THREAD

**Discussion by unlearnt** [Cyfrin/foundry-full-course-cu#2149] [Q&A]
This should be the correct command 

forge script script/DeploySimpleStorage.s.sol --rpc-url $Sepolia_RPC_URL --private-key $PRIVATE_KEY --broadcast

instead of 

forge script script deploySimpleStorage.s.sol --rpc_url=$Sepolia_RPC_URL --private-key=$private_key --broadcast

**Comment by EngrPips** [comment]
Hello @unlearnt, Thanks for the heads-up.

**Comment by tesla809** [comment]
Works for me! Thanks!

**Comment by EngrPips** [comment]
Awesome.
